README.rst

```diff
@@ -4,18 +4,45 @@ Oculy: Modular data viewer
 Installation
 ------------
 
-Currently Oculy cannot be installed directly due to one of its dependency not
-being available on PyPI. To install it directly from GitHub, you can::
+First, all you need is to clone the repository of oculy on your local machine::
 
-    pip install git+https://github.com/MatthieuDartiailh/glaze
-    pip install git+https://github.com/MatthieuDartiailh/oculy
+    git clone https://github.com/MatthieuDartiailh/oculy.git
 
-Once installed Oculy can be started by either running::
+Now that you cloned oculy, you are to to build your package. You may need to install the build
+package first::
+
+    pip install build
+
+OThis should create a `build/` folder as well as a `dist/` folder
+(don't forget to add these to your `.gitignore`, if they aren't in there already.
+
+you can now build your package by running::
+
+ python -m build --wheel
+
+from the folder where the `pyproject.toml` resides. In `dist/` folder you will find a wheel
+named `oculy-0.0.0-py2.py3-none-any.whl`. That file contains your package, and this is all
+you need to distribute it. You can install this package anywhere by copying it to the relevant
+machine and running::
+
+    pip install oculy-0.0.0-py2.py3-none-any.whl
```

Comment:
Technically you can simply run `pip install .` . Users have little interest in generating the intermediate files.

Reply:
Do you mean `pip install oculy-0.0.0-py2.py3-none-any.whl` and `pip install .` are equivalent?

Reply:
No I mean that pip install . from the root of the repo will build the package and install it all at once without having to manually build the package which is a bit cumbersome for users.

Reply:
> No I mean that pip install . from the root of the repo will build the package and install it all at once without having to manually build the package which is a bit cumbersome for users.

👍